spec/files/docs/example.csv

```diff
@@ -1,3 +1,3 @@
-slug,title,description,price
-onion,Onion,Red Big,86
-potato,Potato,Sweet Small,242
+slug,title,description,price,delivery_method

```

Comment:
I would suggest renaming this file, so it describes what it is containing. What is the difference between those two files btw?

Reply:
Fixed.
This is for testing external data for import rewards from .csv file. The first one checks the import and adding the data to DB; the second contains the updated data and checks if the data is updated correctly.